imsv-docs-docusaurus/package.json

```diff
@@ -21,8 +21,8 @@
     "@mdx-js/react": "^3.0.1",
     "clsx": "^2.1.1",
     "docusaurus-lunr-search": "^3.4.0",
-    "docusaurus-plugin-openapi-docs": "^3.0.0-beta.10",
-    "docusaurus-theme-openapi-docs": "^3.0.0-beta.10",
+    "docusaurus-plugin-openapi-docs": "^3.0.1",
+    "docusaurus-theme-openapi-docs": "^3.0.1",
```

Comment:
this was a required change to get the build to work, I presume we didn't encounter this earlier, as we weren't adding new sidebar items